pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -3,12 +3,15 @@ package openflow
 import (
 	"errors"
 	"fmt"
+	"strconv"
 	"sync"
 	"time"
 
+	"k8s.io/klog"
+
 	"github.com/contiv/libOpenflow/openflow13"
 	"github.com/contiv/ofnet/ofctrl"
-	"k8s.io/klog"
+	"github.com/vmware-tanzu/antrea/pkg/agent/metrics"
```

Comment:
We group imports by builtin, 3rd, antrea's own, instead of packages's domain, e.g. github.com.
So they should be:
```
"github.com/contiv/libOpenflow/openflow13"
"github.com/contiv/ofnet/ofctrl"
"k8s.io/klog"

"github.com/vmware-tanzu/antrea/pkg/agent/metrics"
```